packages/storybook/src/components/accordion/accordion.css-template.ts

```diff
@@ -53,14 +53,22 @@ export const cssAccordion: ComponentImplementation<Accordion> = {
 
       if (section.handleUrl) {
         return html`
-          <a href="${section.handleUrl}" aria-expanded=${ariaExpanded}>
+          <a
+            href="${section.handleUrl}"
+            class=${accordion.reverseAlign ? 'dso-right-align' : 'dso-left-align'}
```

Comment:
Nu is dit een breaking change geworden voor het html/css component. De afnemer moet deze classes gaan plaatsen.

Ik denk dat dit niet nodig is omdat op `div.dso-accordion` een class komt als reverse align aan staat. `div.dso-accordion.dso-accordion-reverse-align`